packages/db/src/translate/file.ts

```diff
@@ -1,16 +1,21 @@
 import { ApiFile } from '@cumulus/types/api/files';
+import { ValidationError } from '@cumulus/errors';
 
 import { PostgresFile } from '../types/file';
 
 export const translateApiFiletoPostgresFile = (
   file: ApiFile
-): Omit<PostgresFile, 'granule_cumulus_id'> => ({
-  bucket: file.bucket,
-  checksum_type: file.checksumType,
-  checksum_value: file.checksum,
-  file_name: file.fileName,
-  file_size: file.size,
-  key: file.key,
-  path: file.path,
-  source: file.source,
-});
+): Omit<PostgresFile, 'granule_cumulus_id'> => {
+  if (!file.bucket || !file.key) throw new ValidationError(`bucket and key properties are required: ${JSON.stringify(file)}`);
```

Comment:
I don't think we need or want this. This is essentially replicating the enforcement of the schema from the database in code. I think it is preferable to let the database enforce that on write and not have to duplicate the logic in code

Reply:
Right, we talked about this recently. Taking a look at removing it and leaning on the Postgres schema.

Reply:
Updated here: https://github.com/nasa/cumulus/pull/2129/commits/bf8eff5425d57c2c3b944ae7fce6b078ef29e050